svc-aca-py/build.gradle

```diff
@@ -14,3 +14,10 @@ task buildImage(type: DockerBuildImage, group: 'docker') {
 pushBuildImage {
 	images.add(svcAcaPyImageFullName)
 }
+
+task printImageFullName {
```

Comment:
Do we need this gradle task?

Reply:
Yes, right now this is missing and breaking the container check for AcaPy:
https://github.com/SSI-Solutions/vcms/blob/main/.github/workflows/container-scan.yml#L204 

This use-case is a bit conflicted, the Acapy image is not created by us, so we cannot really fix issues directly in it. I suggest to fix the container checking and see what the results show. It might be helpful feedback to move quicker to newer AcaPy releases. If it ends up as just spam, we can just drop the check and in general aim to keep it up-to-date.